commit ab8e1a87f608fb0fc6a88639150a652b17f424ef
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Fri Mar 10 02:39:25 2017 +0100

    mv resourcebase.kcfg -> pim4resourcebase.kcfg
    
    Resolves a file collision with KF5Akonadi

diff --git a/akonadi/CMakeLists.txt b/akonadi/CMakeLists.txt
index 1d32b3f92..3bd0c8290 100644
--- a/akonadi/CMakeLists.txt
+++ b/akonadi/CMakeLists.txt
@@ -261,7 +261,7 @@ qt4_add_dbus_adaptor( akonadikde_LIB_SRC interfaces/org.freedesktop.Akonadi.Agen
 
 
 kde4_add_kcfg_files( akonadikde_LIB_SRC resourcebasesettings.kcfgc )
-install( FILES resourcebase.kcfg DESTINATION ${KCFG_INSTALL_DIR} )
+install( FILES pim4resourcebase.kcfg DESTINATION ${KCFG_INSTALL_DIR} )
 
 kde4_add_ui_files( akonadikde_LIB_SRC
   cachepolicypage.ui
diff --git a/akonadi/resourcebase.kcfg b/akonadi/pim4resourcebase.kcfg
similarity index 100%
rename from akonadi/resourcebase.kcfg
rename to akonadi/pim4resourcebase.kcfg
diff --git a/akonadi/resourcebasesettings.kcfgc b/akonadi/resourcebasesettings.kcfgc
index 685637b55..df025beac 100644
--- a/akonadi/resourcebasesettings.kcfgc
+++ b/akonadi/resourcebasesettings.kcfgc
@@ -1,4 +1,4 @@
-File=resourcebase.kcfg
+File=pim4resourcebase.kcfg
 ClassName=ResourceBaseSettings
 NameSpace=Akonadi
 Singleton=true
